lib/apruve/resources/order.rb

```diff
@@ -54,7 +54,6 @@ def initialize(params)
     def validate
       errors = []
       errors << 'merchant_id must be set' if merchant_id.nil?
-      errors << 'payment_term must be supplied' if payment_term.nil?
```

Comment:
These are client-side validations, intended to help someone build an Order to pass to the into the checkout Javascript.  In those scenarios, payment_term details won't be known until the checkout completes.  We have to wait to verify them until the Order is sent to the server.